src/AlgebraicGeometry/RationalPoint/AffineRationalPoint.jl

```diff
@@ -220,3 +241,9 @@ function decide_du_val_singularity(P::AbsAffineRationalPoint{<:FieldElem,<:Any})
   d = decide_du_val_singularity(codomain(P), ideal(P))
   return d[1][1],d[1][3]
 end
+
+function stalk(X::AbsSpec, P::AbsAffineRationalPoint)
```

Comment:
The notion of stalk is defined for sheaves, not for spectra. You want the stalk of the structure sheaf here. Is this accepted abuse of notation?

Reply:
I would say so. Formally a scheme X is a ringed space (X, O_X) one just drops O_X from notation. 
And we do not have O_X .

Reply:
Indeed, I agree with this argument.